HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5606,6 +5835,486 @@ def self.get_ceiling_fan_months(weather)
+        next if heat_pump.fraction_cool_load_served == 0
+
+        watts += HVAC.get_dx_cooling_coil_power_watts_from_capacity(UnitConversions.convert(heat_pump.cooling_capacity, 'btu/hr', 'kbtu/hr'))
+        watts += HVAC.get_blower_fan_power_watts(heat_pump.fan_watts_per_cfm, heat_pump.cooling_airflow_cfm)
+      end
+
+    elsif type == HPXML::ElectricPanelLoadTypeWaterHeater
+      hpxml_bldg.water_heating_systems.each do |water_heating_system|
+        next if !system_ids.include?(water_heating_system.id)
+        next if water_heating_system.fuel_type != HPXML::FuelTypeElectricity
+        next if water_heating_system.is_shared_system
+
+        if water_heating_system.water_heater_type == HPXML::WaterHeaterTypeStorage
+          watts += UnitConversions.convert(water_heating_system.heating_capacity, 'btu/hr', 'w')
+        elsif water_heating_system.water_heater_type == HPXML::WaterHeaterTypeHeatPump
+          watts += [UnitConversions.convert(Waterheater.get_heating_input_capacity(water_heating_system.heating_capacity, water_heating_system.additional_properties.cop), 'btu/hr', 'w'),
+                    UnitConversions.convert(water_heating_system.backup_heating_capacity, 'btu/hr', 'w')].max
+        elsif water_heating_system.water_heater_type == HPXML::WaterHeaterTypeTankless
+          if hpxml_bldg.building_construction.number_of_bathrooms == 1
+            watts += default_panels_csv_data['wh_tankless1'][voltage]['PowerRating']
+          elsif hpxml_bldg.building_construction.number_of_bathrooms == 2
+            watts += default_panels_csv_data['wh_tankless2'][voltage]['PowerRating']
+          else # 3+
+            watts += default_panels_csv_data['wh_tankless3'][voltage]['PowerRating']
+          end
+        end
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeClothesDryer
+      hpxml_bldg.clothes_dryers.each do |clothes_dryer|
+        next if !system_ids.include?(clothes_dryer.id)
+        next if clothes_dryer.fuel_type != HPXML::FuelTypeElectricity
+
+        if clothes_dryer.is_vented
+          watts += default_panels_csv_data['dryer'][voltage]['PowerRating']
+        else # HP
+          watts += default_panels_csv_data['dryer_hp'][voltage]['PowerRating']
+        end
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeDishwasher
+      hpxml_bldg.dishwashers.each do |dishwasher|
+        next if !system_ids.include?(dishwasher.id)
+
+        if voltage == HPXML::ElectricPanelVoltage240
+          runner.registerWarning("PanelLoad '#{type}' has PanelLoad/Voltage (#{voltage}) not specified in default_panels.csv; PanelLoad/PowerRating will be defaulted.")
+          default_panels_csv_data['dishwasher'][voltage] = {}
+          default_panels_csv_data['dishwasher'][voltage]['PowerRating'] = default_panels_csv_data['dishwasher'][HPXML::ElectricPanelVoltage120]['PowerRating']
+        end
+        watts += default_panels_csv_data['dishwasher'][voltage]['PowerRating']
+      end
```

Comment:
Still a little confused right in here. We enter this block when:
* there is a dishwasher
* the power rating was not specified

The defaults table specifies power rating and breaker spaces for a 120V dishwasher. What if a user specified a 240V dishwasher? We should assume the power rating of a 120V dishwasher, and throw a warning?